x/programs/runtime/runtime.go

```diff
@@ -58,6 +58,10 @@ func NewRuntime(
 	return runtime
 }
 
+func (r *WasmRuntime) WithDefaults(callInfo *CallInfo) CallContext {
```

Comment:
Wondering if it would be a good practice to add a `nil` check here for the pointer